jobs/package-dockertested/install-origin-release.sh

```diff
@@ -15,6 +15,17 @@ docker tag openshift/origin-docker-registry:latest "openshift/origin-docker-regi
 
 cd /data/src/github.com/openshift/aos-cd-jobs
 playbook_base='/usr/share/ansible/openshift-ansible/playbooks/'
+
+ansible-playbook -vv --become               \
+                 --become-user root         \
+                 --connection local         \
+                 --inventory sjb/inventory/ \
+                 -e deployment_type=origin  \
+                 -e openshift_pkg_version="$( cat ./ORIGIN_PKG_VERSION )"               \
+                 -e oreg_url='openshift/origin-${component}:'"$( cat ./ORIGIN_COMMIT )" \
+                 -e openshift_disable_check=docker_image_availability,package_update,package_availability    \
+                 ${playbook_base}prerequisites.yml
+
 if [[ -s "${playbook_base}/openshift-node/network_manager.yml" ]]; then
     playbook="${playbook_base}openshift-node/network_manager.yml"
```

Comment:
Can you add a `/` here after `"${playbook_base}"`? Looks like it may be an unrelated bug? Or actually remove the `/` from the `if [[ -s` check?